src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import mate.academy.spring.validation.Email;
+
 public class UserRequestDto {
+    @Email
     private String email;
+    @Min(6)
     private String password;
```

Comment:
what about repeted password?

Reply:
As I understood its an optional task

Reply:
Ok. We can leave it to the mentor rewiev